packages/example/src/components/PanEvent/panEventHandler.ts

```diff
@@ -0,0 +1,86 @@
+import { HWEvent } from 'react-native';
+import RemoteControlManager from '../remote-control/RemoteControlManager';
+import { SupportedKeys } from '../remote-control/SupportedKeys';
+import { PanEventInterface } from './PanEvent.interface';
+import { throttle } from '../../utils/throttle';
+import { repeat } from '../../utils/repeat';
+
+const GRID_SIZE = 1920;
+const NUMBER_OF_COLUMNS = 5;
+
+const PanEvent: PanEventInterface = {
```

Comment:
why do we declare the PanEventInterface if it's used only once? we could let typescript infer, right?

Reply:
also, I wonder if a class (with a singleton instance...) wouldn't be a little clearer to show that this is holding a state?